### ISSUE TITLE
pypi's pplpy page needs updating

### ISSUE THREAD

**Issue by dimpase** [sagemath/pplpy#12]
it still points not here, but to GitLab's page.

**Comment by mkoeppe** [comment]
This is updated by changing the pplpy source code and pushing a new release.

**Comment by videlec** [comment]
There is a bunch of occurrences
```
$ git grep gitlab
README.rst:You can find the latest version of the source code on gitlab:
README.rst:https://gitlab.com/videlec/pplpy
docs/source/index.rst:The code source can be obtained from `gitlab <https://gitlab.com/videlec/pplpy>`_::
docs/source/index.rst:    $ git clone https://gitlab.com/videlec/pplpy.git
setup.cfg:url = https://gitlab.com/videlec/pplpy
```

**Comment by dimpase** [comment]
What is holding up a new release, besides this? Yes, I still have to look at my CI wheelbuilding issue/PR on gitlab.

**Comment by videlec** [comment]
fixed in #16
